docs/1.getting-started/12.upgrade.md

```diff
@@ -11,10 +11,22 @@ navigation.icon: i-ph-arrow-circle-up-duotone
 
 To upgrade Nuxt to the [latest release](https://github.com/nuxt/nuxt/releases), use the `nuxi upgrade` command.
 
-```bash [Terminal]
+::code-group
+
+```bash [npx]
 npx nuxi upgrade
 ```
 
+```bash [pnpm]
+pnpm dlx nuxi upgrade
```

Comment:
```suggestion
pnpx nuxi upgrade
```

Should be a shorthand IIRC

Reply:
`pnpx` is deprecated now and `pnpm dlx` is the right command (unfortunately)

Reply:
Didn't know - apparently so is `npx` instead of `npm exec` (or they plan to)

Reply:
Sure, why not 👍🏻

(Don't have a strong opinion about bunx or bun x tbh)

Reply:
> Sure, why not 👍🏻
> 
> (Don't have a strong opinion about bunx or bun x tbh)

I'm having some trouble running locally is it okay to ask you, and if so is this an appropriate place?
In both my laptop and my pc at home. I've set the nuxt.com docs in the env to be the docs folder of my nuxt repo, but I'm still getting some errors.

Reply:
I got it working on my linux machine

> > Sure, why not 👍🏻
> > (Don't have a strong opinion about bunx or bun x tbh)
> 
> I'm having some trouble running locally is it okay to ask you, and if so is this an appropriate place? In both my laptop and my pc at home. I've set the nuxt.com docs in the env to be the docs folder of my nuxt repo, but I'm still getting some errors.

So what would be more appropriate, just have this pr for upgrade.md? Or have all the changes in the one PR?

Reply:
Change it everywhere if you want to ☺️